### PR TITLE
docs(jaml): add missing docstrings

### PR DESCRIPTION
**Pull request by Yongxuanzhang**

### CODE REVIEW

**Review by theUnkownName — DISMISSED** [review]
LGTM!

File: jina/jaml/parsers/default/v1.py [hunk 1]
```diff
@@ -5,6 +5,7 @@
 
 
 class V1Parser(VersionedYAMLParser):
+    """V1 default parser, used as the default parser for objects except driver, executor and flow."""
```
Comment (JoanFM) [inline jina/jaml/parsers/default/v1.py:8]: Are we sure is not used for Flow if version is set to 1?

**Review by JoanFM — CHANGES_REQUESTED** [review]
Could you check the questions I asked? You can add prints and load yamls of different objects to see if they are called for different objects

File: jina/jaml/__init__.py [hunk 2]
```diff
@@ -105,6 +105,15 @@ def load_no_tags(stream, **kwargs):
     def expand_dict(d: Dict, context: Union[Dict, SimpleNamespace, None] = None,
                     resolve_cycle_ref=True,
                     resolve_passes: int = 3) -> Dict[str, Any]:
+        """
+        Expand variables from YAML file.
+
+        :param d: flow yaml file loaded as python dict
```
[the remark this answers is not in the document]
Comment (Yongxuanzhang) [reply jina/jaml/__init__.py:111]: I think you're correct, 'flow' is not correct. I cannot remember why I add 'flow' here.

File: jina/jaml/parsers/default/v1.py (hunk 1 above)
Comment (Yongxuanzhang) [reply jina/jaml/parsers/default/v1.py:8]: To my understanding, this is the V1 default parser, and we have another v1 parser for flow. 
image

Comment (Yongxuanzhang) [reply jina/jaml/parsers/default/v1.py]: I updated the expression in the new commit.

File: jina/jaml/helper.py [hunk 3]
```diff
@@ -33,6 +39,13 @@ def get_hashable_key(self, key):
         return key
 
     def construct_mapping(self, node, deep=True):
+        """
+        Build the mapping between
+
+        :param node: the node to traverse
+        :param deep:
```
Comment (theUnkownName) [inline jina/jaml/helper.py:46]: Can you add more info to this parameters?

File: jina/jaml/helper.py [hunk 4]
```diff
@@ -33,6 +39,13 @@ def get_hashable_key(self, key):
...
+        :return:
```
Comment (theUnkownName) [inline jina/jaml/helper.py:47]: Also maybe add info for the return type

**Review by theUnkownName — CHANGES_REQUESTED** [review]
Add more info to parameteres

File: jina/jaml/helper.py (hunk 3 above)
Comment (Yongxuanzhang) [reply jina/jaml/helper.py]: 👍

File: jina/jaml/helper.py (hunk 4 above)
Comment (Yongxuanzhang) [reply jina/jaml/helper.py]: 👍

**Review by theUnkownName — APPROVED** [review]
Great job!

**Pull request merged**